### PR TITLE
rationalize charm-tools section in charms-deploying

### LINKED ISSUE #2101: point to charm tools page in charms-deploying
Taken from #2092

### PR DESCRIPTION
**Pull request by pmatulis**
Fixes #2101

### CODE REVIEW

File: src/en/tools-charm-tools.md [hunk 1]
```diff
@@ -2,21 +2,20 @@ Title: Charm tools
...
+The 'Charm Tools' Juju Plugin is a collection of commands enabling users and
+charm authors to create, search, fetch, update, and manage charms.
 
 # Installation
 
 The source project can be found at
 [https://launchpad.net/charm-tools](https://launchpad.net/charm-tools).
 
-## Ubuntu 16.04 LTS (Xenial) and later releases
+## Ubuntu
```
Comment (evilnick) [inline src/en/tools-charm-tools.md]: What about other OSes? Perhaps this should borrow the language from the Ubuntu install page and have another heading for CentOS and other Linuxes

File: src/en/charms-deploying.md [hunk 2]
```diff
@@ -367,55 +367,17 @@ official Charm Store. Such cases include:
 - The charms may exist online but they have been customized locally.
 
 !!! Note: 
-    Although this method will ensure that the charms themselves are
-    available on systems without outside internet access, there is no guarantee
-    that a charm will work in a disconnected state. Some charms will attempt to pull
-    code from sources on the internet such as GitHub.
+    Although this method will ensure that the charms themselves are available
```
Comment (evilnick) [inline src/en/charms-deploying.md]: Can you turn off your 'auto-wrapping' setting or whatever - it just generates noise in the diffs

Comment (pmatulis) [reply src/en/charms-deploying.md]: I don't understand.

File: src/en/tools-charm-tools.md (hunk 1 above)
Comment (pmatulis) [reply src/en/tools-charm-tools.md]: OK.

**Pull request merged**